From c500ce2c25278f411cac1d71ed73d7e3fcae5e42 Mon Sep 17 00:00:00 2001
From: Pascal Paillet <p.paillet@st.com>
Date: Tue, 19 Mar 2024 17:39:15 +0100
Subject: [PATCH 0708/1695] dts: stm32: fix wake-up from Standby for stm32mp15
 boards

Fix wake-up from Standby for stm32mp15 boards.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 6d81e43f189c2a7a5b6bdab4549bd49c5903ba48 dts: stm32: fix wake-up from Standby for stm32mp15 boards]
Change-Id: I87a59cc57c82da6b2b6b4a1b4253bb1102e28b42
Signed-off-by: Pascal Paillet <p.paillet@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/367955
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/arch/arm/dts/stm32mp157a-ed1.dts  | 2 +-
 core/arch/arm/dts/stm32mp157c-ed1.dts  | 2 +-
 core/arch/arm/dts/stm32mp157d-ed1.dts  | 2 +-
 core/arch/arm/dts/stm32mp157f-ed1.dts  | 2 +-
 core/arch/arm/dts/stm32mp15xx-dkx.dtsi | 2 +-
 5 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp157a-ed1.dts b/core/arch/arm/dts/stm32mp157a-ed1.dts
index 268f81d58..ce5da7871 100644
--- a/core/arch/arm/dts/stm32mp157a-ed1.dts
+++ b/core/arch/arm/dts/stm32mp157a-ed1.dts
@@ -179,7 +179,7 @@
 				regulator-name = "vdd_sd";
 				regulator-min-microvolt = <2900000>;
 				regulator-max-microvolt = <2900000>;
-				regulator-boot-on;
+				st,regulator-warm-boot-on;
 			};
 
 			v1v8: ldo6 {
diff --git a/core/arch/arm/dts/stm32mp157c-ed1.dts b/core/arch/arm/dts/stm32mp157c-ed1.dts
index b8a54c748..a6593b909 100644
--- a/core/arch/arm/dts/stm32mp157c-ed1.dts
+++ b/core/arch/arm/dts/stm32mp157c-ed1.dts
@@ -302,7 +302,7 @@
 				regulator-min-microvolt = <2900000>;
 				regulator-max-microvolt = <2900000>;
 				interrupts = <IT_CURLIM_LDO5 0>;
-				regulator-boot-on;
+				st,regulator-warm-boot-on;
 			};
 
 			v1v8: ldo6 {
diff --git a/core/arch/arm/dts/stm32mp157d-ed1.dts b/core/arch/arm/dts/stm32mp157d-ed1.dts
index e707e6694..0a207a6fb 100644
--- a/core/arch/arm/dts/stm32mp157d-ed1.dts
+++ b/core/arch/arm/dts/stm32mp157d-ed1.dts
@@ -178,7 +178,7 @@
 				regulator-name = "vdd_sd";
 				regulator-min-microvolt = <2900000>;
 				regulator-max-microvolt = <2900000>;
-				regulator-boot-on;
+				st,regulator-warm-boot-on;
 			};
 
 			v1v8: ldo6 {
diff --git a/core/arch/arm/dts/stm32mp157f-ed1.dts b/core/arch/arm/dts/stm32mp157f-ed1.dts
index 062cf1eb8..881608f78 100644
--- a/core/arch/arm/dts/stm32mp157f-ed1.dts
+++ b/core/arch/arm/dts/stm32mp157f-ed1.dts
@@ -184,7 +184,7 @@
 				regulator-name = "vdd_sd";
 				regulator-min-microvolt = <2900000>;
 				regulator-max-microvolt = <2900000>;
-				regulator-boot-on;
+				st,regulator-warm-boot-on;
 			};
 
 			v1v8: ldo6 {
diff --git a/core/arch/arm/dts/stm32mp15xx-dkx.dtsi b/core/arch/arm/dts/stm32mp15xx-dkx.dtsi
index 7ac634607..76649c2c0 100644
--- a/core/arch/arm/dts/stm32mp15xx-dkx.dtsi
+++ b/core/arch/arm/dts/stm32mp15xx-dkx.dtsi
@@ -401,7 +401,7 @@
 				regulator-min-microvolt = <2900000>;
 				regulator-max-microvolt = <2900000>;
 				interrupts = <IT_CURLIM_LDO5 0>;
-				regulator-boot-on;
+				st,regulator-warm-boot-on;
 			};
 
 			v1v2_hdmi: ldo6 {
-- 
2.47.2

